docs/aspnet.md

```diff
@@ -158,6 +158,24 @@ services.AddMvc().AddFluentValidation(fv => {
 
 Note that if you enable this behaviour you should not use `SetValidator` for child properties, or the validator will be executed twice.
 
+### Implicit Validation of Collection-Type Models
+
+By default, you must create a specific collection validator or enable implicit child property validation to validate a model that is of a collection type. For example, no validation of the following model will occur with the default settings unless you define a validator that inherits from `AbstractValidator<List<Person>>`.
+
+```csharp
+public ActionResult DoSomething(List<Person> people) => Ok();
```

Comment:
I changed this example from `ICollection` to `List` for clarity, as by default validating with interface types is slightly misleading. When you specify an interface collection type as a controller parameter, MVC will internally create a `List<T>`, and as such the validator must be defined as `AbstractValidator<List<T>>` not `AbstractValidator<ICollection<T>>`. It's always better to be explicit with controller parameters, as using interface collection types here 'hides' the magic.